cub/device/device_segmented_radix_sort.cuh

```diff
@@ -95,6 +98,9 @@ struct DeviceSegmentedRadixSort
    *   `[d_end_offsets, d_end_offsets + num_segments)` in any way.
    * - @devicestorageNP For sorting using only `O(P)` temporary storage, see 
    *   the sorting interface using DoubleBuffer wrappers below.
+   * - Segments are not required to span `num_items`. Ranges of `d_keys_out` 
+   *   and `d_values_out` that are not covered by segments are guaranteed not 
+   *   to be modified. 
```

Comment:
For your consideration, here's two options for extending my previous comment to make it more specific to this function. Pick whichever is your favorite (or something all together different!):


Option 1
```
 * @par Segments are not required to be contiguous. For all index values `i` outside the specified segments 
 * `d_keys_in[i], d_values_in[i], d_keys_out[i], d_values_out[i]` will not be accessed nor modified. 
```

Option 2
```
 * @par Segments are not required to be contiguous. Elements of `d_keys_in, d_values_in, d_keys_out, d_values_out` 
 * outside the specified segments  will not be accessed nor modified. 
```

Reply:
I'm assuming that whatever you decide will be reflected in the docs for the other functions.